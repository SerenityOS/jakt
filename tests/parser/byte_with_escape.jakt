function main() {
    let x = b'\n'
    let y = b'\"'
    let z = b'\''

    println("{} {} {}", x, y, z)
}