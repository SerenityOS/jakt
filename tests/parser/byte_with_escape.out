10 34 39
